docs/attributes-registry/installation.md

```diff
@@ -0,0 +1,36 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Installation
+
+## Installation Attributes
+
+Describes installation attributes.
+
+| Attribute | Type | Description | Examples | Stability |
+|---|---|---|---|---|
+| <a id="installation-id" href="#installation-id">`installation.id`</a> | string | A unique identifier representing the installation of an application on a specific device [1] | `identifier_for_vendor:2ab2916d-a51f-4ac8-80ee-45ac31a28092`; `firebase:2ab2916d-a51f-4ac8-80ee-45ac31a28092`; `guid:2ab2916d-a51f-4ac8-80ee-45ac31a28092`; `app_set_id:2ab2916d-a51f-4ac8-80ee-45ac31a28092`; `android_id:2ab2916d-a51f-4ac8-80ee-45ac31a28092` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1] `installation.id`:** Its value MUST remain the same between launches of the same installation of an application.
+On a particular device it MIGHT be the same across different applications of the same vendor, however, it MUST be different across applications of different vendors.
+It SHOULD change if the application is uninstalled or if all applications of the vendor are uninstalled.
```

Comment:
If an application is uninstalled and then re-installed later, I'm guessing it should have different `installation.id` for the second install. So I'm curious why we're using "SHOULD" here?

Reply:
My interpretation of [`Settings.Secure.ANDROID_ID`](https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID) is that it survives uninstalls and re-installs:

> On Android 8.0 (API level 26) and higher versions of the platform, a 64-bit number (expressed as a hexadecimal string), unique to each combination of app-signing key, user, and device. Values of ANDROID_ID are scoped by signing key and user. The value may change if a factory reset is performed on the device or if an APK signing key changes.

So if we want to allow this to be used as `installation.id`, I think we cannot use "MUST" here.

Reply:
Got it, thanks. I think is fine to leave SHOULD here.